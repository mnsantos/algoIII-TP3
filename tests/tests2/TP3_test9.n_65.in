65 670
2 10
1 8
1 2
3 17
1 7
2 8
1 3
7 9
4 17
2 7
3 11
8 10
2 3
6 7
1 4
4 11
9 13
7 8
7 12
1 16
1 17
1 18
5 15
1 9
6 13
1 6
1 12
1 5
2 16
1 10
5 12
5 7
2 18
6 8
3 4
6 11
8 11
1 11
7 13
10 12
64 65
62 65
62 64
62 63
61 64
60 65
59 62
58 65
58 64
58 62
57 62
57 59
55 63
55 62
55 61
55 60
54 65
54 59
54 56
54 55
53 64
53 61
53 56
53 54
51 58
51 57
51 55
50 63
50 61
50 60
50 52
49 65
49 60
49 56
48 65
48 64
48 63
48 58
48 54
48 52
48 51
47 61
47 59
47 54
47 53
47 49
46 65
46 57
46 50
46 48
45 65
45 62
45 57
45 55
45 54
44 63
44 61
44 58
44 50
44 49
44 48
43 64
43 63
43 61
43 60
43 54
43 50
43 49
43 48
42 65
42 60
42 58
42 56
42 48
42 44
41 61
41 56
41 54
41 48
41 43
40 65
40 63
40 62
40 57
40 55
40 53
40 51
40 45
40 41
39 64
39 59
39 57
39 55
39 53
39 51
39 49
39 42
39 40
38 64
38 62
38 60
38 54
38 50
38 48
38 45
38 44
38 40
37 63
37 59
37 58
37 55
37 54
37 51
37 48
37 46
37 42
37 41
36 65
36 63
36 61
36 58
36 55
36 49
36 43
36 42
36 39
35 63
35 61
35 58
35 53
35 50
35 49
35 47
35 39
35 37
34 64
34 63
34 61
34 60
34 57
34 56
34 55
34 52
34 51
34 50
34 49
34 48
34 47
34 43
34 39
34 37
34 36
33 65
33 63
33 59
33 58
33 53
33 51
33 47
33 43
33 42
33 41
33 40
33 39
33 35
32 64
32 53
32 51
32 50
32 46
32 43
32 39
32 37
31 62
31 50
31 49
31 48
31 44
31 43
31 39
31 35
30 64
30 58
30 55
30 54
30 53
30 51
30 46
30 45
30 43
30 42
29 64
29 61
29 52
29 50
29 49
29 41
29 39
28 62
28 58
28 54
28 52
28 47
28 41
28 38
28 34
28 30
27 65
27 64
27 56
27 55
27 49
27 47
27 33
27 29
26 64
26 56
26 54
26 52
26 45
26 44
26 42
26 37
26 36
26 34
26 32
25 63
25 55
25 54
25 53
25 47
25 46
25 44
25 42
25 41
25 35
24 63
24 62
24 60
24 58
24 57
24 55
24 50
24 49
24 47
24 45
24 43
24 41
24 38
24 33
24 29
24 25
23 64
23 62
23 61
23 58
23 52
23 50
23 42
23 41
23 40
23 35
23 32
23 25
23 24
22 58
22 57
22 56
22 55
22 52
22 51
22 48
22 46
22 44
22 42
22 37
22 35
22 26
22 25
22 24
22 23
21 63
21 54
21 52
21 51
9 36
9 32
9 30
9 25
9 24
9 23
9 20
9 17
9 15
9 14
8 63
8 61
8 57
8 56
8 53
8 48
8 41
8 33
8 30
8 29
8 28
8 24
8 21
8 17
8 15
7 61
7 56
7 54
7 51
7 47
7 40
7 38
7 37
7 34
7 27
7 23
7 22
7 18
6 56
6 55
6 54
6 53
6 50
6 48
6 41
6 40
6 36
6 33
6 29
6 27
6 25
6 18
6 17
5 64
5 63
5 62
5 55
5 49
5 28
5 27
5 25
4 55
4 53
4 51
4 49
4 45
4 42
4 36
4 32
4 31
4 29
4 23
4 21
4 18
3 56
3 52
3 51
3 47
3 46
3 45
1 50
1 49
1 48
1 47
1 46
1 45
1 44
1 43
1 42
1 41
1 40
1 39
1 38
1 37
1 36
1 35
1 34
1 33
1 32
1 31
1 30
1 29
1 28
1 27
1 19
1 20
1 21
1 22
1 23
1 24
1 25
1 26
1 51
1 52
1 53
1 54
1 55
1 56
1 57
1 58
1 59
1 60
1 61
1 62
1 63
1 64
1 65
2 20
2 22
2 23
2 25
2 29
2 40
2 42
2 43
2 46
2 49
2 50
2 52
2 53
2 60
2 61
2 63
2 64
3 18
3 19
3 30
3 31
9 37
9 39
9 44
9 55
9 56
9 60
9 64
10 14
10 17
10 18
10 20
10 23
10 25
10 26
10 28
10 34
10 35
10 38
10 41
10 42
10 50
10 53
10 55
10 58
10 62
10 63
10 64
11 13
11 14
11 16
11 23
11 25
11 28
11 39
11 40
11 48
11 49
11 52
11 53
11 57
11 58
11 60
12 20
12 21
12 24
12 34
12 35
12 36
12 39
12 40
12 41
12 45
12 46
12 47
12 52
12 53
12 55
12 58
12 59
12 61
12 65
13 14
13 15
13 20
13 25
13 27
13 32
13 33
13 39
13 44
13 45
13 47
13 50
13 53
13 59
13 60
13 62
14 28
14 30
14 31
14 33
14 39
14 40
14 50
14 54
14 56
14 57
14 60
14 61
14 65
15 18
15 21
15 23
15 26
15 31
15 32
15 35
15 37
15 41
15 42
15 43
15 46
15 48
15 49
15 50
15 51
15 55
15 57
15 61
16 18
16 19
16 20
16 29
16 38
16 44
16 48
16 49
16 52
16 53
16 55
16 59
16 60
16 65
17 19
17 22
17 24
17 27
17 34
17 35
17 37
17 49
17 51
17 56
17 57
17 59
17 60
17 61
17 64
18 21
18 34
18 36
18 40
18 42
18 44
18 49
18 51
18 57
18 58
18 59
18 62
19 23
19 25
19 32
19 34
19 36
19 37
19 40
19 42
19 48
19 54
19 55
19 56
19 58
19 60
19 65
20 22
20 28
20 30
20 36
20 38
20 51
20 53
20 55
20 60
20 61
20 64
20 65
21 22
21 28
21 31
21 32
21 33
21 35
21 39
21 40
21 41
21 43
21 45
21 46
21 48
21 50
1 15
1 13
2 12
2 14
1 14
2 5
0
