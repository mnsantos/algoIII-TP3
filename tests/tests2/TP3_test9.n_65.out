154 6 1 12 34 39 36 55
